### ISSUE TITLE
Missing detailed info on overlay numbers on left toolbar-buttons

### ISSUE THREAD

**Issue by hoppfrosch** [Komodo/KomodoEdit#1545]
### Short Summary

On the toolbar on the left (see screenshot) there are overlay icons with numbers. What do they mean? Especially for Sourcecode-Control its not clear what the number means. You cannot find any information about it on the whole GUI 

### Steps to Reproduce
Starting situation: Currently the overlay icon on "Source Code Control" button shows the number 5
Hovering over the "Source Code Control" button shows a tooltip with content "Source Code Control" - left clicking on the button opens up a menu, containing the Source Code Control commands, Switching to the bottom pane tab "Version Control" and going through the separate sub-tabs does not reflect any information which could be correlated with the "5" as an overlay ...

### Expected results
"Syntax Check" button on the left toolbar has a "1" as overlay icon - left clicking on this button opens a menu, shows an entry concerning this single syntax-checking issue. Clicking on this entry jumps to the causing line in the sourcecode.
I would expect something similar for "Source Code Control" button (listing the "meaning" of the indicated 5 items - for example if the 5 indicates, that there are currently 5 changed files - the names of the files should be listed in the menu and on clicking the menu the corresponding files should be opened)

I would even prefer if the "issues" are not listed within the menu but in the toolbar which occurs when hovering the button

### Platform Information

Komodo IDE, version 10.0.1, build 89160, platform win32-x86.
Win7 SP1 Ultimate 64-bit 

### Additional Information

![komodo_2016-05-19_10-13-59](https://cloud.githubusercontent.com/assets/419075/15387279/92d16304-1dac-11e6-92c4-92a169a8ef47.png)

**Comment by mitchell-as** [comment]
Yes, I agree it is not clear what this number means. Thanks for bringing this up!

**Comment by Naatan** [comment]
Thanks @hoppfrosch, the number on the VCS icon indicates the number of files that changed. I do not believe it currently counts "new" (untracked) files. 

I'll make this more evident.
